mkt/reviewers/views.py

```diff
@@ -448,17 +444,12 @@ def app_review(request, addon):
         # them to index the app or call update_version() if that wasn't done
         # before already.
         if request.method == 'POST':
-            try:
+            with transaction.atomic():
```

Comment:
you can remove this. The entire view will be under transaction management anyway, and will be rolled back if those queries fail